pkg/agent/endpoints/workload/handler.go

```diff
@@ -44,6 +44,8 @@ type Config struct {
 	Manager                       Manager
 	Attestor                      Attestor
 	AllowUnauthenticatedVerifiers bool
+	AllowedForeignJWTClaims       map[string]bool
```

Comment:
i know the ergonomics are a little nicer with map[string]bool, but I prefer map[string]struct{} for sets for a few reasons:
1 - optimized storage (not very relevant here)
2 - don't have to think about the conditions where the key exists in the map or if the value could somehow be false